client/my-sites/site-settings/form-general.jsx

```diff
@@ -609,10 +617,26 @@ export class SiteSettingsFormGeneral extends Component {
 
 		const LaunchCard = showPreviewLink ? CompactCard : Card;
 
+		if ( isSiteOnECommerceTrial ) {
+			eCommerceTrialUpsell = (
+				<a href={ `/plans/${ siteSlug }?plan=business-bundle-monthly` }>
+					<Notice
+						icon="info"
+						showDismiss={ false }
+						text={ translate(
+							'In order for everyone to see your store, you just need to subscribe to a eCommerce plan. Subscribe'
```

Comment:
I think "Subscribe" would make sense as a `<NoticeAction />` linked to the plans page, to make it more clear this notice is click-able, rather than wrapping the whole component in an anchor.

I know this is following the given design, but the wording here feels off to me. It would be good to get another set of eyes on the copy before we add the "String Freeze" label to get it ready for translation.

Reply:
Agreed on both counts: we should use the `NoticeAction` component for the CTA, and the wording here is a bit awkward.

@verofasulo, could you take a first pass at the wording for us? My thinking is that something like the following would be better:
> You can launch your store by subscribing to a paid eCommerce plan [Subscribe]

or 

> To launch your store, you need to subscribe to a paid eCommerce plan [Subscribe]